packages/rector-generator/src/Node/NodeClassProvider.php

```diff
@@ -14,7 +14,7 @@ final class NodeClassProvider
     public function getNodeClasses(): array
     {
         $robotLoader = new RobotLoader();
-        $robotLoader->addDirectory(__DIR__ . '/../../../../vendor/nikic/php-parser/lib/PhpParser/Node');
+        $robotLoader->addDirectory(__DIR__ . '/../../../../../../nikic/php-parser/lib/PhpParser/Node');
```

Comment:
It is working in the main repository. 
It's probably because it's installed as dependency. The best would be to support both locations.

Reply:
Got it @TomasVotruba 

Should I add condition here with `file_exists()` call?

Reply:
Whatever works